cypress/support/step_definition/room_boards/commonRoomBoardSteps.js

```diff
@@ -0,0 +1,9 @@
+const { When, Then } = require("@badeball/cypress-cucumber-preprocessor");
+import RoomBoards from "../../pages/room_board/pageRoomBoards";
+
+const roomBoards = new RoomBoards();
+
+When("I click outside the BBB dialog box", () => {
+	// this assumes the BBB dialog box is open and to click outside for closing it.
```

Comment:
I don't think we need this comment as its obvious that modal should be opened and visible at that step